python/ucxx/_lib_async/tests/test_send_recv_am.py

```diff
@@ -83,16 +84,14 @@ async def test_send_recv_am(size, recv_wait, data):
             # immediately as receive data is already available.
             await asyncio.sleep(1)
         await c.am_send(msg)
-
-    while len(recv) == 0:
-        await asyncio.sleep(0)
+        recv_msg = await c.am_recv()
```

Comment:
<!-- Polish comments are like a suggestion, where there is nothing necessarily wrong with the relevant content, there's just some ways to immediately improve the quality.  -->
**polish:** ‏This waits for the result from every client, but overwrites with the most recent result. So where previously we checked only the first result, now we check only the last result.

Now, there's only a single client, so we're actually checking all the results. But perhaps we should make this test correct for more than one client. WDYT?

Reply:
Good catch, fixed in https://github.com/rapidsai/ucxx/pull/101/commits/7a0b1c3436e1b8a0cc92dac744312dc2a04228fd .